lib/signaling/v2/room.js

```diff
@@ -122,6 +122,9 @@ class RoomV2 extends RoomSignaling {
       _trackSwitchOffSignaling: {
         value: new options.TrackSwitchOffSignaling(getTrackReceiver, { log }),
       },
+      _otherMediaSignalings: {
```

Comment:
We have used this nomenclature since the beginning, for example TrackSignaling, PeerConnectionSignaling etc. So, I would like to retain the naming convention.